spec/lib/tasks/update_used_aliquots_with_primary_aliquot_tag.rake_spec.rb

```diff
@@ -7,20 +7,28 @@
 Rails.application.load_tasks if Rake::Task.tasks.empty?
 
 RSpec.describe 'RakeTasks' do
+  before do
+    Pacbio::SmrtLinkVersion.find_by(name: 'v13_sequel_iie') || create(:pacbio_smrt_link_version, name: 'v13_sequel_iie', default: true)
+  end
+
   describe 'used_aliquots:update_tags' do
```

Comment:
Might be good to add correct libraries that shouldn't get updated to check that the task doesn't touch them.